docs/patterns/card/index.md

```diff
@@ -160,5 +199,17 @@ A card can be used in light and dark themes.
     padding-inline: 0;
     padding-block: 0;
   }
+
+  rh-card.custom-light-theme {
+    --rh-color-border-subtle-on-light: #EF6461;
+    --rh-color-surface-lightest: #0081AF;
```

Comment:
I suggest making the surface color lighter. Maybe use `#feeded`? It's a custom color used in the homepage hero.

Reply:
Awesome, thanks!